extension/lib/main.js

```diff
@@ -113,14 +110,65 @@ exports.main = function (options, callbacks) {
         widget.tooltip = config.extension.widget_tooltips[data];
         break;
     }
+  });*/
+
+  var { Frame } = require("sdk/ui/frame");
+  var { ToggleButton } = require('sdk/ui/button/toggle');
+  var { Toolbar } = require("sdk/ui/toolbar");
+
+  var frame = new Frame({
+    url: "./frame/frame.html"
   });
 
+  var menuButton = ToggleButton({
+    id: "menu-button",
```

Comment:
> I would give it the id `memchaser-toolbar-menu-button` so that it is clear where it belongs to.

I'm unsure what do you want from the `memchaser-toolbar-` prefix.

This id should be specific in addon context only.
The SDK will generate a longer id for the real button, like `toggle-button--memchaserqualitymozillaorg-menu-button`.

If you want to show that this button belongs to the toolbar, and not the other one from #176, then I'll rename it.